tests/urql/atomWithQuery.test.tsx

```diff
@@ -26,7 +26,7 @@ const generateClient = (id: string | number, error?: () => boolean) =>
         fromValue(
           error?.() ? { error: new Error('fetch error') } : { data: { id } }
         ),
-        delay(100)
+        delay(100) // FIXME we want to use fake timer
```

Comment:
@kitten Can you quickly guide how we can control this flow in tests?

Reply:
Sorry that I didn't see this in time!

I'm not sure why it wouldn't use the fake timers as long as the global is modified in time 🤔 https://github.com/0no-co/wonka/blob/cc88647f65d0e24d84d2715b7af4eac4dc9df01f/src/operators.ts#L800-L817 That's at least true for the new major of Wonka.

Reply:
Yeah, we could have used fake timers. (I did in other tests.)

From the URQL user perspective, I don't want to reach wonka. I just need a fake/mock client, in which case, fake timers are more appropriate. If you have a suggestion, it would be more than welcomed.